be/src/exec/tablet_sink.h

```diff
@@ -332,25 +332,30 @@ class IndexChannel {
     OlapTableSink* _parent;
     int64_t _index_id;
 
-    // from backend channel to tablet_id
-    // ATTN: must be placed before `_node_channels` and `_channels_by_tablet`.
-    // Because the destruct order of objects is opposite to the creation order.
-    // So NodeChannel will be destructured first.
-    // And the destructor function of NodeChannel waits for all RPCs to finish.
-    // This ensures that it is safe to use `_tablets_by_channel` in the callback function for the end of the RPC.
+    // Members order here is error prone, DO NOT change it unless you are
```

Comment:
I didn't get it. in #8501 and #8607 I add a `_closed_lock` in NodeChannel so that if the NodeChannel has been closed, it won't call `mark_as_failure` anymore.

Reply:
the _closed_lock fixes the problem, my code is not updated.